python/sglang/srt/layers/quantization/modelopt_quant.py

```diff
@@ -35,10 +35,20 @@
     from sglang.srt.layers.moe.topk import TopKOutput
 
 if is_cuda():
-    from sgl_kernel import cutlass_scaled_fp4_mm, scaled_fp4_quant
+    from sgl_kernel import scaled_fp4_quant
+
+try:
```

Comment:
nit: would be great to have a env var to allow user to control which to use, thus if one gets some bug we have another